### ISSUE TITLE
16.6.0 crashes on my Fire TV when in standby

### ISSUE THREAD

**Issue by taalas** [koying/SPMC#873]
Since upgrading to 16.6.0 SPMC seems to crash when the Fire TV goes into standby. I notice this because I get an error message stating this when turning on my TV set and Fire TV the next day.

**Comment by JFG90** [comment]
Are you not exiting spmc?

**Comment by taalas** [comment]
No, I normally use the back button to return to the main UI. I was under the impression that running in the background was a feature of SPMC?

**Comment by JFG90** [comment]
When you are done using SPMC you should always exit properly, or it can cause all sorts of problems for example it crashing

**Comment by taalas** [comment]
This confuses me greatly to be honest. The standard Kodi will exit if I press the back button going back to the Fire TV UI, SPMC behaves differently. Why does it implement this differently if I am to shut it down after use?

**Comment by JFG90** [comment]
I have no idea but like anything if you have finished with it you should shut it down

**Comment by JFG90** [comment]
I have no idea but like anything if you have finished with it you should shut it down

**Comment by rbronco21** [comment]
Why is SPMC Launcher available if you can't use it as the launcher? I've never closed SPMC or Kodi with launcher intent, but I have noticed some weird behavior with 16.6.0 that may be related.

**Comment by unlocKing** [comment]
I'm having same issue on Fire TV products, when coming back to it after I while I see it has crashed.

**Comment by taalas** [comment]
@JonnyGadd While your statement may be true for some software products, I don't really see it applying here tbh.

**Comment by JFG90** [comment]
Whatever, excuse me for trying to help sort it out on your own.

**Comment by JFG90** [comment]
I apologise if that's come across rude, but I've only been trying to help

**Comment by taalas** [comment]
I am thankful for your attempt at helping and my answer wasn't meant to be rude either. I just wanted to state that I don't really agree. The fact that the software crashes and didn't do so before seems to indicate a problem to me and I wanted to report that.

**Comment by JFG90** [comment]
That's fair enough, hope you get it sorted

**Comment by rbronco21** [comment]
What's happening? No unbridled aggression, just civil discussion? You've let the internets down... For real, though, thanks for making this a place for normal people. And thanks for any help you have to give.
